tsl/src/bgw_policy/drop_chunks_api.c

```diff
@@ -55,6 +58,137 @@
 										  Int32GetDatum(0),                                        \
 										  Float8GetDatum(0)))
 
+/* TODO:: Q:: These should probably be in its own file? */
+Datum
+ts_integer_from_now_func_get_datum(int64 interval, Oid time_dim_type, Oid now_func)
```

Comment:
Should these be in a separate file or is here good?

Reply:
yes its own file would be better

Reply:
ping